CONTRIBUTING.md

```diff
@@ -24,16 +24,15 @@ If you are interested in contributing to ESPnet, your contributions will fall in
    (e.g., see https://github.com/espnet/espnet/blob/master/egs/tedlium2/asr1/RESULTS.md).
    
    To make a report for `RESULTS.md`
-	 - execute `get_sys_info.sh` at a recipe main directory (where `run.sh` is located), as follows. 
-	   You'll get environmental information in a markdown format.
+	 - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
+	   You'll get environmental information and the evaluation result of each experiments in a markdown format.
 	   ```
-	   $ get_sys_info.sh
+	   $ show_result.sh
 	   ```
 	 - execute `pack_model.sh` at a recipe main directory as follows. You'll get model information and results in a markdown format
```

Comment:
You'll get model information and results in a markdown format
->
You'll get model information in a markdown format